clusterloader2/pkg/framework/client/objects.go

```diff
@@ -41,6 +41,10 @@ const (
 	retryBackoffFactor          = 3
 	retryBackoffJitter          = 0
 	retryBackoffSteps           = 6
+
+	// Default time parameters
```

Comment:
Maybe sth more informative, like `Parameters for namespace deletion operations.`?